Classes/Database/RteImagesSoftReferenceIndex.php

```diff
@@ -70,6 +76,22 @@ public function findRef($table, $field, $uid, $content, $spKey, $spParams, $stru
         return $retVal;
     }
 
+    public function parse(string $table, string $field, int $uid, string $content, string $structurePath = ''): SoftReferenceParserResult
+    {
+        // does nothing
+        return SoftReferenceParserResult::createWithoutMatches();
```

Comment:
findRef code should be moved here and return a parser result to restore functionality.